WcaOnRails/app/views/avatars_mailer/notify_user_of_avatar_removal.html.erb

```diff
@@ -0,0 +1,12 @@
+<p>Hello <%= @user.name %></p>
```

Comment:
Shouldn't there be a comma at the end of this line?

Reply:
@danieljames-dj resolved in 0b2f9e39bedd207f183104d5b7a6151b79560a43🚀